contents/blog/posthog-vs-fathom.mdx

```diff
@@ -0,0 +1,178 @@
+  <ComparisonRow column1={true} column2={false} feature="Open source" description="View code, comment on roadmap, and contribute" />
+  <ComparisonRow column1={true} column2={false} feature="Heatmaps" description="Visualize where users click in your app or website" />
+  <ComparisonRow column1={true} column2={false} feature="Session replay" description="Watch real users use your product, diagnose bugs" />
+  <ComparisonRow column1={true} column2={false} feature="Experiments" description="Run A/B/n tests on new features, optimize conversion funnels" />
+  <ComparisonRow column1={true} column2={false} feature="Surveys" description="Collect and analyze feedback, run NPS and PMF surveys" />
+  <ComparisonRow column1={true} column2={false} feature="Feature flags" description="Roll out features safely, toggle features for cohorts or individuals" />
+  <ComparisonRow column1={true} column2={false} feature="Server-side SDKs" description="Capture events and use features from Python, Node, and more" />
+  <ComparisonRow column1={true} column2={false} feature="Mobile SDKs" description="Capture events and use features from Android, iOS, and more" />
+</ComparisonTable>
+
+## Integrations
+
+<ComparisonTable column1="PostHog" column2="Fathom">
+  <ComparisonRow column1={true} column2={true} feature="CSV exports" description="Export your data as a CSV" />
+  <ComparisonRow column1={true} column2={true} feature="Email" description="Send reports to email" />
+  <ComparisonRow column1={true} column2={false} feature="Warehouse exports" description="Export data to warehouses like S3 and BigQuery for storage and analysis" />
+  <ComparisonRow column1={true} column2={false} feature="Data warehouse" description="Query customer and product data together" />
+  <ComparisonRow column1={true} column2={false} feature="Warehouse import" description="Import data from third-party sources like Postgres, S3, and GCS" />
+  <ComparisonRow column1="Events, reports" column2="Reports, anomalies" feature="Slack" description="Send events, reports, and anomalies to Slack" />
+  <ComparisonRow column1={true} column2={false} feature="Stripe" description="Import charges, customers, invoices, subscriptions, and more" />
+  <ComparisonRow column1={true} column2={false} feature="HubSpot" description="Import contacts, companies, deals, and more" />
+  <ComparisonRow column1={true} column2={false} feature="Zapier" description="Trigger Zapier automations" />
+  <ComparisonRow column1={true} column2={false} feature="Zendesk" description="Import organizations, tickets, users, and more" />
+  <ComparisonRow column1={true} column2={false} feature="Sentry" description="Send and receive error data with Sentry" />
+  <ComparisonRow column1={true} column2={true} feature="WordPress" description="Send and receive error data with Sentry" />
```

Comment:
Although we can install on Wordpress, I wouldn't say we have an integration. Maybe with changing + adding a note?
```suggestion
  <ComparisonRow column1={true} column2={true} feature="WordPress" description="Easily capture data from your Wordpress site" />
```